packages/matrix-client-server/src/user/account_data/index.ts

```diff
@@ -0,0 +1,98 @@
+import type MatrixClientServer from '../..'
+import {
+  jsonContent,
+  validateParameters,
+  errMsg,
+  type expressAppHandler,
+  send
+} from '@twake/utils'
+
+interface parameters {
+  userId: string
+  type: string
+}
+
+interface PutRequestBody {
+  content: string
+}
+
+const schema = {
+  content: true
+}
+
+const mxidRe = /^@[0-9a-zA-Z._=-]+:[0-9a-zA-Z.-]+$/
```

Comment:
Maybe a more explicit name here. 
Also we shall document in a comment values matched by this regex
Last but not least we should beware of Regular Expression denial of service because of backtracking. I suggest a quick fuzzing to ensure this is not the case. Also using a regexp engine without backtracting works to.

Reply:
After thinking about it I believe this regex pattern shouldn't cause denial of service cause of backtracking since it should only perform a linear scan on the given matrixId to see if each character belongs to the given set before and after ":". Will fix the name though, thank you !